.github/workflows/release.yml

```diff
@@ -27,3 +27,16 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           AUR_SSH_KEY: ${{ secrets.AUR_SSH_KEY }}
           HOMEBREW_TAP_GITHUB_TOKEN: ${{ secrets.HOMEBREW_TAP_GITHUB_TOKEN }}
+
+      - name: Generate release tarball
+        run: |
+          VERSION=${GITHUB_REF_NAME#v}
+          tar czf antidot-${VERSION}.tar.gz --transform "s,^,antidot-${VERSION}/," --exclude dist *
+
+      - name: Upload release tarball
+        uses: softprops/action-gh-release@master
```

Comment:
I prefer to pin actions to a certain version. I saw [v0.1.14](https://github.com/softprops/action-gh-release/tree/v0.1.14) is the latest version of this action. Could you change to it?

Reply:
fixed